node-red-contrib-bot-message/bot-send-card.js

```diff
@@ -552,38 +562,40 @@ const buildReplyformAdaptiveCard = (RED, node, locale, data, config, reply) => {
     let submitLabelform = config.submitLabelformAdaptiveCard;
     let placeHolderform = config.placeHolderformAdaptiveCard;
 
-    if(config.sendType === 'formAdaptiveCard'){
     formtitle = helper.getContextValue(RED, node, data, formtitle || '', config.titleformTypeAdaptiveCard || 'str');
-    formtitle = marshall(locale, formtitle,  data, '');
-    
-    
-    reply.type = 'AdaptiveCard';
-    reply.formtitle = formtitle;
+    formtitle = marshall(locale, formtitle, data, '');
+
+
+    reply.type = CARD_CONST.CARD_ADAPTIVECARD;
+    reply.title = formtitle;
     reply.version = "1.3";
-    reply.body = [{"type": "TextBlock",
-    "size": "Medium",
-    "weight": "Bolder",
-    "text": formtitle,
-    "horizontalAlignment": "Center",
-    "wrap": true},
-    {
-        "type": "Input.Text",
-        "style": "text",
-        "isMultiline": true,
-        "id": "userFeedback",
-        "placeholder": placeHolderform,
-        "maxLength": 500,
-        "isRequired": true,
-        "label": "Feedback:",
-        "errorMessage": errorMsgform
-    }];
+    reply.body = [{
+            "type": "TextBlock",
+            "size": "Medium",
+            "weight": "Bolder",
+            "text": formtitle,
+            "horizontalAlignment": "Center",
+            "wrap": true
+        },
+        {
+            "type": "Input.Text",
+            "style": "text",
+            "isMultiline": true,
+            "id": "userFeedback",
+            "placeholder": placeHolderform,
+            "maxLength": 500,
+            "isRequired": true,
+            "label": "Feedback:",
```

Comment:
line 588, the label `feedback` should be extracted to the configuration, the same way as `placeHolderform`